src/components/ui/HeaderDialog.jsx

```diff
@@ -0,0 +1,63 @@
+'use client'
+
+import { Dialog } from '@headlessui/react';
+import { XMarkIcon } from '@heroicons/react/24/outline';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useMobileMenuContext } from '../../app/HeaderContext';
+
+const HeaderDialog = ({ data: links, company, logIn }) => {
+  const { mobileMenu, closeMobileMenu } = useMobileMenuContext();
+
+  return (
+    <Dialog as="div" className="lg:hidden" open={mobileMenu} onClose={closeMobileMenu}>
+    <div className="fixed inset-0 z-10" />
+    <Dialog.Panel className="fixed inset-y-0 right-0 z-10 w-full overflow-y-auto bg-white px-6 py-6 sm:max-w-sm sm:ring-1 sm:ring-gray-900/10">
+      <div className="flex items-center justify-between">
+        <Link href="/" className="-m-1.5 p-1.5">
+          <span className="sr-only">{company.name}</span>
+          <Image 
+            src={`${company.logo.url}`}
+            alt="logo"
+            width="40"
+            height="40"
+          />
+        </Link>
+        <button
+          type="button"
+          className="-m-2.5 rounded-md p-2.5 text-gray-700"
+          onClick={closeMobileMenu}
+        >
+          <span className="sr-only">Close menu</span>
+          <XMarkIcon className="h-6 w-6" aria-hidden="true" />
+        </button>
+      </div>
+      <div className="mt-6 flow-root">
+        <div className="-my-6 divide-y divide-gray-500/10">
+          <div className="space-y-2 py-6">
+            {links.links.map(link => (
```

Comment:
links.links? something not right with naming here

Reply:
fixed